### PR TITLE
Wire up drive

### PR DESCRIPTION
**Pull request by QuantumOtterDev**
This resolves ticket #20

### CODE REVIEW

File: src/main/java/frc/robot/operatorInputs/OperatorInputs.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.robot.subsystems.drive.Drive;
+
+
+
+public class OperatorInputs {
+
+  private final double JOYSTICK_DEADZONE = 0.1;
+
+  public OperatorInputs(Controls driverControls, Controls operatorControls, Drive drive) {
```
Comment (kushagharahi) [inline src/main/java/frc/robot/operatorInputs/OperatorInputs.java]:
```suggestion
  public OperatorInputs(Controls driverControls, Drive drive) {
```

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -16,8 +22,13 @@
  */
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
-
+  private Drive drive;
+  private Gyroscope gyro;
   private RobotContainer m_robotContainer;
+  private Controls driverControls;
```
Comment (kushagharahi) [inline src/main/java/frc/robot/Robot.java:27]: Add a `OperatorInputs` object here and "wire" it up in the `robotInit()` method like we've done for the `drive`, `gyro`, and `driverControls` objects. Let me know if this is confusing and we will work through it

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -4,9 +4,15 @@
...
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.commands.drivingCommands.ArcadeDriveCommand;
+import frc.robot.subsystems.drive.Drive;
+import frc.robot.subsystems.drive.Gyroscope;
+import frc.robot.operatorInputs.Controls;
```
Comment (kushagharahi) [inline src/main/java/frc/robot/Robot.java:13]: Make sure all these imports in green are being used when you open it up in vscode. They'll be grayed out if they are not being used.

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -4,9 +4,14 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
```
Comment (kushagharahi) [inline src/main/java/frc/robot/Robot.java:7]: Is joystick needed here?

Comment (EganJ) [reply src/main/java/frc/robot/Robot.java:7]: Looks like the constructor for Controls requires it

**Review by EganJ — APPROVED** [review]
Successfully tested on the robot.

**Review by kushagharahi — APPROVED** [review]
Approved. Make sure everything is pushed up and merge

**Pull request merged**